ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+- [ ] I'm reporting a bug, not asking for help
+- [ ] I've looked at the [documentation](http://leafletjs.com/reference-1.0.0.html) to make sure this is not a documented behaviour
+- [ ] I'm sure this is a Leaflet code issue, not an issue with my own code
```

Comment:
I'd add a line about ionic, cordova, react, ractive, vue and all the other JS frameworks we don't (really) care about.